src/solanaUtils/solanaLibraryBufferFix.ts

```diff
@@ -0,0 +1,7 @@
+import { Buffer } from "buffer"
+
+// @ts-ignore
+if (typeof window !== "undefined" && typeof window.Buffer === "undefined") {
+// @ts-ignore
+  window.Buffer = Buffer
```

Comment:
Why is this necessary? Based on my understanding, Solana web3.js has been free of `Buffer` since version 2.0 https://github.com/solana-foundation/solana-web3.js/issues/1100#issuecomment-2427708857